app/src/main/res/values/strings.xml

```diff
@@ -468,4 +468,8 @@ like to connect to:</string>
   <string name="sensor_description_active_notification_count">Total count of active notifications that are visible to the user including silent, persistent and the Sensor Worker notifications.</string>
     <string name="basic_sensor_name_app_importance">App Importance</string>
   <string name="sensor_description_app_importance">If the app is in the foreground, background or any other state it can be.</string>
+    <string name="security_vulnerably_title">Security Vulnerably Warning</string>
+  <string name="security_vulnerably_message">Your Home Assistant is out of date and contains a known security vulnerably.  Please view the details via the bulletin and update as soon as possible.</string>
```

Comment:
This text is incorrect. Home Assistant is secure. The latest version of HA adds new protections to protect against vulnerable custom components.

Reply:
It has been changed to

> Your version of Home Assistant does not protect for recently discovered security vulnerabilities in custom components. Please view the details via the bulletin and update as soon as possible.